debian/build.go

```diff
@@ -175,6 +183,38 @@ func walkBuilds(builds []build, f func(pkg, distro, arch string, v version) erro
 	return nil
 }
 
+func fetchVersion(url, fallback string) string {
```

Comment:
fallback seems like a bad idea. If I set this up in CI how do I know whether by pushing the release I would be reverting to an old version? Just fail if we can't fetch a new version.